app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,553 @@
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "The Cabinet",
-              heading_level: 3,
-              margin_bottom: 5,
-            } %>
-
-            <p class="govuk-body">The Cabinet is made up of the senior members of government. Every week during Parliament, members of the Cabinet (Secretaries of State from all departments and some other ministers) meet to discuss the most important issues for the government.</p>
-
-            <p class="govuk-body"><%= link_to("See who is in the Cabinet", "/government/ministers", class: "govuk-link") %></p>
-          </div>
-        </div>
-        <div class="one-half">
-          <div class="inner-block">
-            <%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: 'The Cabinet' %>
-          </div>
-        </div>
-      </div>
+      <section>
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Who runs government",
+          id: "who-runs-government",
+          font_size: "l",
+          margin_bottom: 4,
+        } %>
+        
+        <section class="govuk-!-margin-bottom-9 prime-minister">
+          <%= render "govuk_publishing_components/components/heading", {
+            text: "The Prime Minister",
+            font_size: "m",
+            heading_level: 3,
+            margin_bottom: 4,
+          } %>
+
+          <p class="govuk-body">The <%= link_to("Prime Minister", "/government/ministers/prime-minister", class: "govuk-link") %> is the leader of Her Majesty’s Government and is ultimately responsible for all policy and decisions.</p>
+          <p class="govuk-body">The Prime Minister also:</p>
+
+          <%= render "govuk_publishing_components/components/list", {
+            extra_spacing: true,
+            visible_counters: true,
+            items: [
+              "oversees the operation of the Civil Service and government agencies",
+              "appoints members of the government",
+              "is the principal government figure in the House of Commons",
+            ]
+          } %>
+
+          <% if @prime_minister.present? %>
+            <p class="govuk-body">The Prime Minister is <%= link_to @prime_minister.name, url_for(@prime_minister), class: "govuk-link" %>.</p>
+          <% end %>
+
+          <p class="govuk-body govuk-!-margin-bottom-8">
+            <%= link_to "Read more about the Prime Minister's Office, 10 Downing Street", "/government/organisations/prime-ministers-office-10-downing-street", class: "govuk-link" %>
+          </p>
+          
+          <% if @prime_minister.present? && @prime_minister.image_url(:s465).present? %>
+            <%= image_tag @prime_minister.image_url(:s465), { alt: '', class: 'govuk-!-width-full', width: '465', height: '310' } %>
+          <% else %>
+            <%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: '', class: 'govuk-!-width-full', width: '465', height: '310' %>
```

Comment:
These dimensions need attention - I've pushed your branch onto integration to preview it and the images are being stretched too wide on desktop and pushed too narrow on mobile. They seem ok at tablet size. I'm not sure what the interaction causing that is but it definitely needs a fix before we can merge.

Reply:
Yep I see the issue - I've made a few changes to remove the `govuk-!-width-full` class and have replaced with some fluid image styles -
```
img {
  max-width: 100%;
  height: auto;
}
```
which combined with the dimensions seem to work okay now. i've updated the screenshot in the description and also attached a mobile one here

<img src="https://user-images.githubusercontent.com/87758239/134308604-9a2d34b3-be49-4ac2-894e-f0b1d338e2ad.png" width="300">